addon/usables/use-machine.js

```diff
@@ -7,7 +7,7 @@ import { DEBUG } from '@glimmer/env';
 import { warn } from '@ember/debug';
 
 export const ARGS_STATE_CHANGE_WARNING =
-  'A change to passed `args` or a local state change triggered an update to a `useMachine`-usable. This is currently unsupported and should probably be handled explicitly in your xstate-machine. If you feel this is necessary behavior that should be supported by `ember-statecharts` please raise this as an issue and provide an example of what you are trying to do.';
+  'A change to passed `args` or a local state change triggered an update to a `useMachine`-usable. You can handle this via the `.update`-hook of the `useMachine`-usable.';
```

Comment:
I think adding a bit more info here on how to proceed would help. Something like:

> You can reset the interpreted machine or send a dedicated event so this is handled. This is done via .update ...`